### PR TITLE
Adding default value to the user provided non primary rules

### PR DESCRIPTION
**Pull request by SiniWilson**
### Requirements

Adding default value to the non primary rules

### Description of the Change

Adding default value to the user provided non primary rules
Fixing issues of develop sync with sls-app branch

### Benefits

Templates will pick the default value if the user does not specify any value for mandatory fields

### CODE REVIEW

File: builds/jazz-build-module/sls-app/sbr.groovy [hunk 1]
```diff
@@ -673,7 +683,8 @@ class SBR_To_Constraint implements SBR_Constraint {
   }
 
   public boolean compliant(val) {
-    return val <= toValue;
+    if(val && val !== '')  return Integer.parseInt(val.toString()) <= Integer.parseInt(toValue.toString());
```
Comment (raghits) [inline builds/jazz-build-module/sls-app/sbr.groovy]: Do we need to handle ParseException?

File: builds/jazz-build-module/sls-app/sbr.groovy [hunk 2]
```diff
@@ -685,7 +696,8 @@ class SBR_From_Constraint implements SBR_Constraint {
   }
 
   public boolean compliant(val) {
-     return val >= toValue;
+    if(val && val !== '')  return Integer.parseInt(val.toString()) >= Integer.parseInt(fromValue.toString());
```
Comment (raghits) [inline builds/jazz-build-module/sls-app/sbr.groovy]: Do we need to handle ParseException?

File: builds/jazz-build-module/sls-app/sbr.groovy (hunk 1 above)
Comment (SiniWilson) [reply builds/jazz-build-module/sls-app/sbr.groovy]: Added try catch block

File: builds/jazz-build-module/sls-app/sbr.groovy (hunk 2 above)
Comment (SiniWilson) [reply builds/jazz-build-module/sls-app/sbr.groovy]: Added try catch block

**Pull request merged**